src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.main_header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
+
+.device_nav {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
Be consistent with your *vertical* margins. You have set both `padding: 0;` and `margin: 0;` on `.device_nav`. Choose either padding or margin for vertical spacing to avoid potential collapsing issues.